packages/docs/.vuepress/public/images/anatomy-link.svg

```diff
@@ -1,12 +1,12 @@
08.323 179.784V165.24H404.243V180.744C404.243 186.024 406.595 189.672 412.163 189.672ZM432.141 189H436.221V178.92C436.221 175.416 436.365 168.168 442.893 168.168C446.061 168.168 448.125 169.992 448.125 174.456V189H452.205V173.496C452.205 168.216 449.805 164.568 443.901 164.568C438.765 164.568 436.461 168.216 436.125 169.272V165.24H432.141V189ZM469.394 189.672C474.722 189.672 478.178 186.552 479.714 182.136L475.874 180.792C475.106 183.384 473.33 186.072 469.394 186.072C465.458 186.072 462.386 183.192 462.386 177.096C462.386 171.144 465.554 168.168 469.346 168.168C472.802 168.168 474.53 170.424 475.298 172.68L479.09 171.432C477.986 167.736 474.722 164.568 469.346 164.568C462.77 164.568 458.114 169.752 458.114 177.096C458.114 184.776 462.53 189.672 469.394 189.672ZM485.157 189H489.237V178.92C489.237 175.416 489.381 168.168 495.909 168.168C499.077 168.168 501.141 169.992 501.141 174.456V189H505.221V173.496C505.221 168.216 502.629 164.568 497.205 164.568C491.589 164.568 489.573 168.216 489.237 169.272V164.28V155.4H485.157V189ZM533.248 189.672C537.424 189.672 539.728 187.368 540.88 185.352V189H544.864V155.4H540.784V164.28V168.936C539.632 166.92 537.424 164.568 533.248 164.568C527.776 164.568 523.6 169.32 523.6 177.144C523.6 184.968 527.776 189.672 533.248 189.672ZM534.304 186.072C530.512 186.072 527.776 183.816 527.776 177.192C527.776 170.568 530.512 168.168 534.304 168.168C538.192 168.168 541.024 171.192 541.024 177.192C541.024 183.192 538.192 186.072 534.304 186.072ZM559.082 189.672C562.778 189.672 565.034 188.088 566.81 185.4C567.098 187.368 567.626 189 571.226 189H574.346V185.4H571.85C570.746 185.4 570.458 185.112 570.458 184.008V172.728C570.458 167.832 567.53 164.568 561.194 164.568C555.002 164.568 551.738 167.928 550.874 171.096L554.858 172.536C555.482 170.76 556.778 168.168 561.146 168.168C564.794 168.168 566.378 169.896 566.378 172.776V173.448C558.65 174.072 551.402 176.328 551.402 182.856C551.402 187.08 554.378 189.672 559.082 189.672ZM560.042 186.216C557.066 186.216 555.626 184.776 555.626 182.52C555.626 178.632 560.57 177.144 566.378 176.52V179.16C566.378 184.104 563.162 186.216 560.042 186.216ZM586.647 189H592.119V185.4H587.175C585.495 185.4 584.775 184.68 584.775 183V168.696H591.255V165.24H584.775V157.8H580.695V165.24H575.175V168.696H580.695V183.24C580.695 187.176 582.711 189 586.647 189ZM603.426 189.672C607.122 189.672 609.378 188.088 611.154 185.4C611.442 187.368 611.97 189 615.57 189H618.69V185.4H616.194C615.09 185.4 614.802 185.112 614.802 184.008V172.728C614.802 167.832 611.874 164.568 605.538 164.568C599.346 164.568 596.082 167.928 595.218 171.096L599.202 172.536C599.826 170.76 601.122 168.168 605.49 168.168C609.138 168.168 610.722 169.896 610.722 172.776V173.448C602.994 174.072 595.746 176.328 595.746 182.856C595.746 187.08 598.722 189.672 603.426 189.672ZM604.386 186.216C601.41 186.216 599.97 184.776 599.97 182.52C599.97 178.632 604.914 177.144 610.722 176.52V179.16C610.722 184.104 607.506 186.216 604.386 186.216Z" fill="#1662DD"/>
```

Comment:
What caused these changes? SVGO?

Reply:
These are content changes. Once this PR is merged, I'll double-check optimization in my SVG Opt PR.